proposals/lip-0045.md

```diff
@@ -206,7 +206,7 @@ In this section, we specify the substores that are part of the Interoperability
 | `CCM_STATUS_CODE_FAILED_CCM` | uint32 | 4 | Value of `status` of returned CCM due to error: failed ccm execution. |
 | `CCM_STATUS_CODE_RECOVERED` | uint32 | 5 | Value of `status` of CCM that have been recovered with a message recovery command. |
 | **Message Tags** | | | |
-| `MESSAGE_TAG_CHAIN_REG` | bytes | `“LSK_CHAIN_REGISTRATION_”` ASCII-encoded | Message tag for chain registration message. |
+| `MESSAGE_TAG_CHAIN_REG_MESSAGE` | bytes | `“LSK_CHAIN_REGISTRATION_”` ASCII-encoded | Message tag for chain registration message. |
```

Comment:
Minor comment: I understand that here we want to be consistent with LIP 37, but is it a good naming strategy to have twice the same word in a constant name? Perhaps keeping the old one and updating LIP37 would make more sense, but no strong opinion.

Reply:
The point here is that the first message refers to the fact that this is an encoded value ( we use the message prefix for all of them), the second one to the fact that this is called 'chain registration message' in LIP 43

Reply:
Yep, I understand this.

Reply:
Can I resolve this then? @gkoumout